cmd/prepend.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"errors"
+
+	"github.com/Originate/git-town/lib/git"
+	"github.com/Originate/git-town/lib/prompt"
+	"github.com/Originate/git-town/lib/script"
+	"github.com/Originate/git-town/lib/steps"
+
+	"github.com/spf13/cobra"
+)
+
+type prependConfig struct {
+	InitialBranch string
+	ParentBranch  string
+	TargetBranch  string
+}
+
+var prependCommand = &cobra.Command{
+	Use:   "prepend <branch>",
```

Comment:
hack is just "hack <branch> " and out markdown docs use "prepend <branch_name>". I'll update <branch> to <branch_name> in another PR. I don't think mentioning to create, is necessary as I feel that is captured in the command description.